enhancements/installer/openstack-no-fip-install.md

```diff
@@ -0,0 +1,136 @@
+---
+title: openstack-bring-your-own-external-connectivity
+authors:
+  - "@egarcia"
+  - "@adduarte"
+reviewers:
+  - "@mandre"
+  - "@pierre"
+approvers:
+  - "@mandre"
+  - "@pierre"
+creation-date: 2020-5-26
+last-updated: 2020-5-26
+status: implementable
+---
+
+# OpenStack BYO External Connectivity
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Graduation criteria
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
```

Comment:
Will this also stop creation of the cluster hosted networking components like the haproxy, mdns publisher, keepalived etc.

Reply:
I think the proposed change is only related to topo of underlying openstack created VMs
the container such as haproxy/mdns etc should be almost same to before

Reply:
This proposal is about making the use of an external network (and as a consequence Floating IP) optional. It doesn't change how the networking components are setup.

Reply:
> openstack-no-fip-install.md 

the file name doesn't convey that to me.